views/layout.hbs

```diff
@@ -3,11 +3,37 @@
   <head>
     <title>{{title}}</title>
     <link rel='stylesheet' href='/stylesheets/style.css' />
+    
   </head>
   <body>
-    <form action='/sessions?_method=DELETE' method='POST'>
-      <input id="logout" type="submit" value="Log Out">
-    </form>
+
+    <header>
+      <div class="navbar">
+        <div class="header_buttons">
+          <form action='/' method='GET'>
+            <i class="fa-solid fa-house"></i>
+            <input id="home" class="header_button"  type="submit" value="Home">
+          </form>
+          <form action='/users/new' method='GET'>
+            <input id="sign-up" class="header_button" type="submit" value="Sign Up">
+          </form>
+          <form action='/sessions/new' method='GET'>
+            <input id="log-in"class="header_button"  type="submit" value="Log In">
+          </form>
+          <form action='/sessions?_method=DELETE' method='POST'>
+            <input id="log-out" class="header_button"  type="submit" value="Log Out">
+          </form>
+         </div>
+        </div>
+        </div>
+      
+      
+    </header>
+
     {{{body}}}
+    <script src="https://kit.fontawesome.com/d88bffd7d7.js" crossorigin="anonymous"></script>
```

Comment:
cam i know more about this please? Is this to access custom fonts? If so do we have some backups?

Reply:
This line is for the home icon in the navbar. It is being used as a placeholder until we have chosen to include/not include a home/logo icon to link to the home page. 

It is rendering the icon from an outside website.